src/serde_arrow_array.erl

```diff
@@ -0,0 +1,41 @@
+-module(serde_arrow_array).
+-export([offsets/1]).
+
+-include("serde_arrow_primitive_array.hrl").
+
+-export_type([array/0]).
+-type array() :: #primitive_array{}.
+
+-optional_callbacks([offsets/1]).
```

Comment:
Is there a style guide regarding ordering declarations? Something similar to this [Brew Cask Documentation](https://docs.brew.sh/Cask-Cookbook#stanza-order) would suffice.

Reply:
Not that I know of. What you have is what I tend to do (aside from I put `export_type` after all the `type` declarations).

Reply:
Hmm.. Somebody should write one. It's nice to have some consistency with things like this as a community.